org.mwc.debrief.legacy/src/Debrief/ReaderWriter/GeoPDF/GeoPDFBuilder.java

```diff
@@ -319,13 +343,15 @@ public static File generatePDF(final GeoPDF geoPDF, final GeoPDFConfiguration co
 		while ((line = gdalWarpErrorStream.readLine()) != null) {
 			allOutput.append(line + "\n");
 		}
+		Application.logError3(ToolParent.INFO, "Error generating the PDF: " + allOutput.toString(), null, false);
 		throw new IOException(allOutput.toString());
 	}
 
 	public static GeoPDF build(final Layers layers, final GeoPDFConfiguration configuration)
 			throws IOException, InterruptedException, NoSuchFieldException, SecurityException, IllegalArgumentException,
 			IllegalAccessException, ClassNotFoundException {
 		if (!configuration.isReady()) {
+			Application.logError3(ToolParent.INFO, "GDAL Temporal environment is about to be prepared.", null, false);
```

Comment:
@saulhidalgoaular  -  I think you mean _temporary_, rather than _temporal_ (unless temporal has some specific meaning in this context).